RELEASES.md

```diff
@@ -1,3 +1,116 @@
+- [Add `armv8r-none-eabihf` tier 3 target for the Cortex-R52](https://github.com/rust-lang/rust/pull/110482/)
+- [Rename `wasm32-wasi-preview1-threads` to `wasm32-wasip1-threads`](https://github.com/rust-lang/rust/pull/122170/)
+- [Add a new `wasm32-wasi-preview2` target](https://github.com/rust-lang/rust/pull/119616/)
+
+- [coverage: Split out counter increment sites from BCB node/edge counters](https://github.com/rust-lang/rust/pull/120564/)
+- [coverage: Improve handling of function/closure spans](https://github.com/rust-lang/rust/pull/120569/)
+- [Only generate a ptrtoint in AtomicPtr codegen when absolutely necessary](https://github.com/rust-lang/rust/pull/122220/)
+
+<a id="1.78.0-Libraries"></a>
+
+Libraries
+---------
+- [Optimize away poison guards when std is built with panic=abort](https://github.com/rust-lang/rust/pull/100603/)
+- [PartialEq, PartialOrd: update and synchronize handling of transitive chains](https://github.com/rust-lang/rust/pull/115386/)
+- [Toggle assert_unsafe_precondition in codegen instead of expansion](https://github.com/rust-lang/rust/pull/120594/)
+- [Bump Unicode to version 15.1.0, regenerate tables](https://github.com/rust-lang/rust/pull/120777/)
+- [Implement unwind safety for Condvar on all platforms ](https://github.com/rust-lang/rust/pull/121768/)
+- [Windows: Implement condvar, mutex and rwlock using futex](https://github.com/rust-lang/rust/pull/121956/)
+- [Replace pthread `RwLock` with custom implementation](https://github.com/rust-lang/rust/pull/110211/)
+- [Add `#[rustc_no_mir_inline]` for standard library UB checks](https://github.com/rust-lang/rust/pull/121114/)
+- [Add ASCII fast-path for `char::is_grapheme_extended`](https://github.com/rust-lang/rust/pull/121138/)
+- [Always inline check in `assert_unsafe_precondition` with cfg(debug_assertions)](https://github.com/rust-lang/rust/pull/121196/)
+
+<a id="1.78.0-Stabilized-APIs"></a>
+
+Stabilized APIs
+---------------
+
+- [Add Read Impl for &Stdin](https://github.com/rust-lang/rust/pull/99153/)
+- [`std::error::Error` -> Trait Implementations: lifetimes consistency improvement](https://github.com/rust-lang/rust/pull/113833/)
+- [Make `impl<Fd: AsFd>` impl take `?Sized`](https://github.com/rust-lang/rust/pull/114655/)
+- [impl From<TryReserveError> for `io::Error`](https://github.com/rust-lang/rust/pull/121403/)
+- [Make `Barrier::new()` const](https://github.com/rust-lang/rust/pull/119536/)
+
+<a id="1.78.0-Cargo"></a>
+
+Cargo
+-----
+- [Stabilize lockfile v4](https://github.com/rust-lang/cargo/pull/12852/)
+- [Respect `rust-version` when generating lockfile ](https://github.com/rust-lang/cargo/pull/12861/)
+- [Control `--charset` via auto-detecting config value](https://github.com/rust-lang/cargo/pull/13337/)
+- [Support `target.<triple>.rustdocflags` officially](https://github.com/rust-lang/cargo/pull/13197/)
+- [Allow `--precise` to allow yanked versions in `cargo update`](https://github.com/rust-lang/cargo/pull/13333/)
```

Comment:
```suggestion
- [Allow `--precise` to specify yanked versions in `cargo update`](https://github.com/rust-lang/cargo/pull/13333/)
```
Still could be better but I think it's an improvement nonetheless.

Reply:
It turns out this is still unstable, so we'll just drop it.
```suggestion
```